docs/anomaly-detection/guides/guide-vmanomaly-vmalert.md

```diff
@@ -162,10 +162,11 @@ reader:
 writer:
```

Comment:
the same applies here, add `sampling_period` in reader block and synchronize its value with that in our guide folder